deploy-cloudrun/src/index.ts

```diff
@@ -41,11 +42,11 @@ async function run(): Promise<void> {
 
     // Deploy service
     let serviceResponse = await client.deploy(service);
-    while (!serviceResponse.status!.url) {
-      serviceResponse = await client.getService(service);
+    while (!get(serviceResponse, 'status.url')) {
```

Comment:
Are there any advantages to using `get` instead of  `?.`?

Reply:
`get` does the null and undefined checking for you. Not critical in this instance but started using elsewhere.